### DISCUSSION TITLE
Request: Blink in App Store for Apple silicon Macs

### DISCUSSION THREAD

**Discussion by olofedlund** [blinksh/blink#1627] [Ideas]
Blink 14 is available in the App Store for the Apple silicon Macs. I'd love to be able to use the latest versions.

There are many reasons I want Blink om my Mac. But the two biggest are:
1) I'd like to standardise on using Blink and
2) I need mosh and I'm not pleased with the other alternatives

**Comment by agluck91** [comment]
So as it stands right now, blink can't be a full terminal replacement. I talked in another ticket a little about this.  If you brought blink over, as is, it could definitely act as a supplemental terminal. Kinda like termius. I would use this if it came to the desktop, even without full terminal support. I love how you manage ssh and mosh. It is so seamless

For full desktop terminal support you would need a full shell like zsh and access to all the binary programs not just the blink ones. So it would definitely be a little bit of reachitecfure. 

I think I saw something about you having a web interface for blink? 

Perhaps we can leverage the new web app features in Sonoma. 

I don't know how much popularity this has though because I only see 20 upvotes on the original post. 

😁 You all worked out the kinks with the resizing windows with iPad now too so this really does seem feasible. 

Might need something to manage tabs a little better since we won't have touch support as well.

**Reply by olofedlund** [reply]
No new functionality is needed. It is still fantastic as a remote shell. The mosh support is fantastic. 

The only thing needed is making it available.
